### PR TITLE
Implement `/send`

### LINKED ISSUE #22: Make /send create a task to edit into a larger message
This is a bit of a broad ticket but I don't think there's a good way to break this down further.

### PR DESCRIPTION
**Pull request by A5rocks**
Fixes https://github.com/A5rocks/code-jam-11/issues/22

### CODE REVIEW

**Review by aceeedev — CHANGES_REQUESTED** [review]
bug: it seems when you /send two messages, both messages get slowly added/edited together

example:
/send test message
/send 1234567890
output: test 1m2e3s4s5a6g7e890

**Comment by A5rocks** [comment]
Yeah it should probably be per-user right? Like it adds to a buffer per user?

**Comment by LuckElixir** [comment]
Isn't the combination of messages supposed to be intentional?

**Comment by A5rocks** [comment]
> Isn't the combination of messages supposed to be intentional?

Yes but only across different users; if the same user sends multiple messages they shouldn't multiplex.

File: app/__main__.py [hunk 1]
```diff
@@ -124,7 +125,12 @@ async def on_message(message: discord.Message) -> None:
 @app_commands.describe(message="The message to send")
 async def send(interaction: Interaction, message: str) -> None:
     """Send a message to the current channel."""
-    await interaction.response.send_message(message)
+    if interaction.channel not in interaction.client.database.get_enabled_channels(interaction.guild):
```
Comment (aceeedev) [inline app/__main__.py]: database.get_enabled_channels has been replaced with database.get_channels I believe and also needs to be awaited

File: app/sender.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+
+senders: dict[discord.Channel, Sender] = collections.defaultdict(Sender)
+
+
+async def send(client: DiscordClient, where: discord.Channel, who: discord.User, what: str) -> None:
+    """Add a message to a queue of messages to be sent, potentially starting a new queue."""
+
+    async def cps(p: discord.User) -> float:
+        profile = client.database.get_profile(where.guild, p)
```
Comment (aceeedev) [inline app/sender.py]: database.get_profile needs to be awaited too

**Review by aceeedev — DISMISSED** [review]

**Review by aceeedev — APPROVED** [review]

**Pull request merged**